### ISSUE TITLE
[BUG] [Linter] Error in orphan reporting

### ISSUE THREAD

**Issue by dyve** [Riverside-Healthcare/djLint#164]
- [x] I'm on the [latest version](https://pypi.org/project/djlint/) of djLint
- [x] I've searched the [issues](https://github.com/Riverside-Healthcare/djLint/issues)
- [x] I've read the [docs](https://djlint.com)

## System Info
 - OS: macOS 12.1
 - Python Version: 3.8.12
 - djLint Version: 0.6.9

## Issue

A `th` tag with attributes from a variable is incorrectly reported as an orphan.

## How To Reproduce

Create the template below as `temp.html`

```jinja2
<th {{ attrs }}>
    <a href="{% url %}">{{ content }}</a>
</th>
```

Note that the `th` tag is closed. It gets its attributes from a (simplified) variable. This is not unusual in Django, it is used in the Django forms library for example.

Lint this file with `djlint` and you will get a reponse stating the `th` tag is an orphan.

```
temp.html
───────────────────────────────────────────────────────────────────────────────
H025 3:0 Tag seems to be an orphan. </th>
```

## Configuration

```
[tool.djlint]
# Reference: https://djlint.com/djlint/rules.html
profile = "django"
ignore = "H006,H008,H013,H014,H017,H019,H021,H023,T001,T002,T003"
blank_line_after_tag = "load,extends,endblock,has_perm"
max_line_length = 120
```
